client/localization/elements/sc-page-static/pl.json

```diff
@@ -11,6 +11,15 @@
     "TERMINOLOGY": "TERMINOLOGIA",
     "SIMILES": "PORÓWNANIA",
     "NAMES": "NAZWY",
-    "SUBJECTS": "TEMATY"
+    "SUBJECTS": "TEMATY",
+    "DISCOURSES": "ROZPRAWIAĆ",
+    "VINAYA": "PRAWO MONASTYCZNE",
+    "ABHIDHAMMA": "ABHIDHAMMA",
+    "NUMBERING": "NUMERACJA",
+    "ABBREVIATIONS": "SKRÓTY",
+    "METHODOLOGY": "METODOLOGIA",
+    "ACKNOWLEDGMENTS": "PODZIĘKOWANIE",
+    "LICENSING": "LICENCJONOWNIE",
```

Comment:
LICENCJONOWANIE

Reply:
Oh dear, I guess Google Tranlator failed the test this time!